iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/RemoteReactivePowerControl.java

```diff
@@ -43,4 +43,6 @@ default String getName() {
     RemoteReactivePowerControl setTargetQ(double targetQ);
 
     RemoteReactivePowerControl setEnabled(boolean enabled);
+
+    RemoteReactivePowerControl setRegulatingTerminal(Terminal regulatingTerminals);
```

Comment:
use a default implementation (maybe throwing an exception as 'not implemented')

Reply:
Finally, we decided not to do it